test/Microsoft.Framework.CommonTestUtils/TestUtils.cs

```diff
@@ -9,11 +9,15 @@
 using System.Security.Cryptography;
 using System.Text;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
 
 namespace Microsoft.Framework.CommonTestUtils
 {
     public static class TestUtils
     {
+        public static IRuntimeEnvironment CurrentRuntimeEnvironment = CallContextServiceLocator.Locator.ServiceProvider
```

Comment:
Um... this is not right. The static field can be initialized at any time, maybe even before the service provider